tests/unit/routes/settings/applications-test.js

```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:settings/applications', 'Unit | Route | settings/applications', {
```

Comment:
See the other route test files and ensure this file is like that. (Use the custom `moduleFor` function that is used in the other files instead of the auto-generated one)